install_config/configuring_authentication.adoc

```diff
@@ -173,7 +173,10 @@ openshift_master_identity_providers=[{'name': 'htpasswd_auth', 'login': 'true',
 #openshift_master_request_header_ca_file
 ----
 ====
-
+<1> If you specify your CA certificate location in the
+`openshift_master_identity_providers` parameter, do not specify a certificate
+value in the `openshift_master_ldap_ca` or path in the 
+`openshift_master_ldap_ca_file` parameter.
```

Comment:
Is `openshift_master_ldap_ca` a parameter? If so, s/a certificate value in the `openshift_master_ldap_ca` or/a certificate
value in the `openshift_master_ldap_ca` parameter. Otherwise, LGTM.